installation/docker-containers/high-availability-install/README.md

```diff
@@ -0,0 +1,68 @@
+# Guide to install Rocket.Chat as HA with mongodb replicaset as backend
+
+## Install MongoDB Replicaset
+
+- Use <https://medium.com/@gargar454/deploy-a-mongodb-cluster-in-steps-9-using-docker-49205e231319#.8dfln7n8c> to setup the mongodb replicaset
+- Create oplogger user with rights on local oplog database:
```

Comment:
I'd change to `Create user called oplogger` otherwise seems like talking about a type of user.

For increased clarity might even just change to: `oploguser`